Modernize UI

uniquePWD opened the pull request:
Fixes the syntax error on index.html

uniquePWD committed:
Update history.html

Modernisation

uniquePWD committed:
Merge remote-tracking branch 'origin/patch-2'

uniquePWD committed:
Merge remote-tracking branch 'origin/patch-3'

uniquePWD committed:
Merge remote-tracking branch 'origin/patch-4'

uniquePWD committed:
Merge remote-tracking branch 'origin/patch-5'

uniquePWD committed:
Merge remote-tracking branch 'origin/patch-6'

uniquePWD committed:
Merge remote-tracking branch 'origin/patch-7'

uniquePWD committed:
Merge remote-tracking branch 'origin/patch-8'

uniquePWD committed:
Merge remote-tracking branch 'origin/patch-9'

uniquePWD committed:
Merge remote-tracking branch 'origin/patch-10'

uniquePWD committed:
Merge remote-tracking branch 'origin/patch-11'

uniquePWD committed:
Merge remote-tracking branch 'origin/patch-12'

uniquePWD committed:
Merge remote-tracking branch 'origin/patch-13'

uniquePWD committed:
Merge remote-tracking branch 'origin/patch-14'

uniquePWD committed:
Merge remote-tracking branch 'origin/patch-15'

uniquePWD committed:
Merge remote-tracking branch 'origin/patch-16'

uniquePWD committed:
Merge remote-tracking branch 'origin/patch-17'

uniquePWD committed:
Merge remote-tracking branch 'origin/patch-18'

uniquePWD committed:
Merge remote-tracking branch 'origin/patch-19'

uniquePWD committed:
Merge branch 'rembo10:modernize-ui' into modernize-ui

uniquePWD committed:
Update index.html

Removed the comments that caused the syntax error

uniquePWD commented:
It's only the index.html that's been updated

Jyers commented:
I'm still seeing the following error when running this:

<img width="1865" height="776" alt="image" src="https://github.com/user-attachments/assets/5041c328-68f9-4e53-9027-bb1dd200bd99" />

uniquePWD commented:
> I'm still seeing the following error when running this:
> <img alt="image" width="1865" height="776" src="https://private-user-images.githubusercontent.com/76993396/476373428-5041c328-68f9-4e53-9027-bb1dd200bd99.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTQ4NDc2NTIsIm5iZiI6MTc1NDg0NzM1MiwicGF0aCI6Ii83Njk5MzM5Ni80NzYzNzM0MjgtNTA0MWMzMjgtNjhmOS00ZTUzLTkwMjctYmIxZGQyMDBiZDk5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA4MTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwODEwVDE3MzU1MlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTIxYTA4ODQzZjA2MTE0MTdmYmMwNWY5MGRiNjE1YjNhNzUzYzhlYmFmYjY2NDYxZTE4YzBmYTVkYThmMDQwZWUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.L5LAsgWFiG8OU5LxM6Jm0ROroJU5JJupnqCFdekOEHM">

Honestly, I messed up. I should've sync's my fork first, so the admin or merging the now deleted branches wasn't included in the PR.

From what I'm seeing of your screenshot though, you're pulling the original PR rather than the changes made today. The issue with the original PR is that it's got some python comments in the html, so I removed them all together.

As for the python syntax errors, I haven't even started on those. Ideally someone more proficient with python would run point on those.

confor commented:
i'm currently running headphones inside a ~docker~ podman container. do you happen to know how i can integrate this pull request into the container so i can help test and review it?
